src/cluster.js

```diff
@@ -223,6 +227,11 @@ Cluster.prototype.exists = function(gaxOptions, callback) {
  *
  * @param {object} [gaxOptions] Request configuration options, outlined here:
  *     https://googleapis.github.io/gax-nodejs/CallSettings.html.
+ * @param {function} callback The callback function.
+ * @param {?error} callback.err An error returned while making this
+ *     request.
+ * @param {object} callback.metadata The metadata.
```

Comment:
I believe we can remove this line, per the code:

```js
callback(err, err ? null : self, metadata);
```

Technically, `apiResponse` and `metadata` are the same here. I would vote to keep `apiResponse` as the chosen name for it, as that would be consistent with the other documentation.